react/react1/week2/todo-list/src/components/ToDoItem.js

```diff
@@ -0,0 +1,17 @@
+export const ToDoItem = (props) => {
+    function changeTaskStatus(e) {
+      if (e.target.checked === true) {
+        e.target.previousElementSibling.className = "done";
+      } else {
+        e.target.previousElementSibling.className = "notDone";
+      }
+    }
+  
+    return (
+      <li className="toDoItem" key={props.todo.id} id={props.todo.id}>
+        <label className="notDone">{props.todo.description}</label>
```

Comment:
<label className={toDoStatus?"done":"notDone"}>{props.todo.description}</label>
And You need to create separate CSS file with these two class names 
.done{
    text-decoration: line-through;
}

.notDone{

}